modules/ossm-cr-techPreview.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-reference-smcp.adoc
+
+[id="ossm-cr-techPreview_{context}"]
+= techPreview parameters
+
+The `techPreview` parameter enables early access to features that are in Technology Preview.
```

Comment:
in other places we prefix `spec.`, maybe we should also do `spec.techPreview` here then?